src/mqtt/index.ts

```diff
@@ -84,31 +80,32 @@ export class Mqtt {
 
   /**
    * Reconnects to MQTT server
-   */
+   *//**
   public async reconnect() {
     if (this.client && this.client.connected) {
       await this.disconnect();
     }
 
     return this.connect();
-  }
+  }**/
 
   /**
    * Connects to the MQTT server
    */
   public async connect() {
     await this.waitConnecting();
-    return await this.doConnect();
+    await this.doConnect();
+    this.connected = true;
   }
 
   /**
    * Returns whether client is connected
    *
    * @returns whether client is connected
-   */
+   */ /**
   public isConnected() {
     return this.client?.connected || false;
-  }
+  } */
```

Comment:
Why is this commented out?

Reply:
I believe as part of the upgrade Antti replaced the original "mqtt" import with "paho-mqtt" the comments are left from the original mqtt library. WIll remove comments now as if needed they can be found in the muisti-management repo.